features/step_definitions/team_steps.rb

```diff
@@ -0,0 +1,27 @@
+require 'uri'
+require 'cgi'
+require File.expand_path(File.join(File.dirname(__FILE__), "..", "support", "paths"))
+require File.expand_path(File.join(File.dirname(__FILE__), "..", "support", "selectors"))
+
+module WithinHelpers
+  def with_scope(locator)
+    locator ? within(*selector_for(locator)) { yield } : yield
+  end
+end
+World(WithinHelpers)
+
+When(/^I go create a team/) do
```

Comment:
These step definitions are already defined but generically in the web_steps.rb file. That way we don't have to hardcode the button names etc.

We can add the page in paths.rb and simply use the web_steps versions for smaller code files